docs/userguide/manage_files.rst

```diff
@@ -0,0 +1,177 @@
+
+When a user clicks on the link, if they have the proper access and permissions, Parsec will open and point to the specific file.
+
+In order to get a link to a file, either right-click on a file and select ``Copy link``, or select a file and click ``Copy link`` in the action bar. This will copy the file link that you can share with other users of the workspace (in an email for example).
+
+.. image:: screens/files/share_link.png
+    :align: center
+    :alt: share file link screenshot
+
+File explorer integration
+-------------------------
+
+When you're logged-in to Parsec, your workspaces are available from the file explorer so you can access your data directly from your computer.
+
+Each workspace is exposed in a specific folder matching the workspace name. You can add, delete, rename or open files, or any other other operations that you would normally do, as if your workspace was any regular folder on your computer.
+
+Any operation performed in the file explorer will be automatically synchronized in Parsec.
+
+If you logout or quit Parsec, your workspaces will no longer be accessible from the file explorer, until you log back in.
+
+.. image:: screens/files/file_explorer.png
+    :align: center
+    :alt: file explorer screenshot
+
+Opening a file
+--------------
+
+When you open a file in Parsec, it will be downloaded and stored locally so files that you open often will not have to be downloaded again. Large files may take some time to open the first time for this reason. If you are offline, only the files that have been downloaded while you were online can be opened.
+
+Using an external application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you double-click on a file, it will be opened with the default application that is associated with the file type on your system. This is therefore defined by your system and not by Parsec.
+
+Using Parsec preview
+^^^^^^^^^^^^^^^^^^^^
+
+Some file types can be displayed directly in Parsec without using an external application.
+
+If the file type is supported (see below), Parsec will display a preview of it. While this preview does not allow you to make any change, and may lack some of the functionality provided by external applications, it is the more secure way of viewing your files as they will not leave Parsec.
+
+.. image:: screens/files/pdf_viewer.png
+    :align: center
+    :alt: PDF viewer screenshot
+
+Parsec preview is limited to files with size inferior to 15MB. The following file formats are supported:
+
+* PDF
+* DOCX (Microsoft Word). XLSX (Microsoft Excel) support is coming soon.
+* Audio
+* Images
+* Plain Text
+
+Parsec preview can be disabled in the desktop application settings.
+
+.. image:: screens/files/disable_viewers.png
+    :align: center
+    :alt: Disable viewers screenshot
+    :width: 400
+
+Recent files
+^^^^^^^^^^^^
```

Comment:
The screenshot shows "Recent documents" we should use the same title here (or change it in the app, as you prefer)
```suggestion
Recent documents
^^^^^^^^^^^^^^^^
```